README.md

```diff
@@ -20,124 +22,47 @@ subtype of `DynamicTable`, where each row corresponds to a different event type.
 Unlike for the other event types, users can add their own custom columns to annotate each event type or event time.
 This can be useful for storing event metadata related to data preprocessing and analysis, such as marking bad events.
 
-This extension was developed by Ryan Ly, Ben Dichter, Oliver Rübel, and Andrew Tritt. Information about the rationale,
-background, and alternative approaches to this extension can be found here:
+This extension was developed by Ryan Ly, Oliver Rübel, and the NWB Technical Advisory Board.
+Information about the rationale, background, and alternative approaches to this extension can be found here:
 https://docs.google.com/document/d/1qcsjyFVX9oI_746RdMoDdmQPu940s0YtDjb1en1Xtdw
 
 ## Installation
-Python:  
-```
-pip install ndx-events
-```
-Matlab:  
+Python:
+```bash
+pip install -U ndx-events
 ```
+
+Matlab:
+```matlab
 generateExtension('<directory path>/ndx-events/spec/ndx-events.namespace.yaml');
 ```
-## Example usage
```

Comment:
@rly where the examples moved somewhere else? I found them useful when using this extension.

Reply:
It seems that they were availble in this commit but they did not make it to the end of the PR:

https://github.com/rly/ndx-events/pull/9/commits/0672b9a9e8450ff3589a797d8e65f6969bb5eee5

Is this a mistake?